pkg/sdk/testint/schemas_integration_test.go

```diff
@@ -103,12 +118,122 @@ func TestInt_SchemasCreate(t *testing.T) {
+
+		schema, err := client.Schemas.ShowByID(ctx, schemaId)
+		require.NoError(t, err)
+		assert.Equal(t, schemaId.Name(), schema.Name)
+		assert.Equal(t, comment, schema.Comment)
+
+		params := testClientHelper().Parameter.ShowSchemaParameters(t, schemaId)
+		assertParameterEquals := func(t *testing.T, parameterName sdk.AccountParameter, expected string) {
+			t.Helper()
+			assert.Equal(t, expected, helpers.FindParameter(t, params, parameterName).Value)
+		}
+
+		assertParameterEquals(t, sdk.AccountParameterDataRetentionTimeInDays, "0")
+		assertParameterEquals(t, sdk.AccountParameterMaxDataExtensionTimeInDays, "10")
+		assertParameterEquals(t, sdk.AccountParameterDefaultDDLCollation, "en_US")
+		assertParameterEquals(t, sdk.AccountParameterExternalVolume, externalVolume.Name())
+		assertParameterEquals(t, sdk.AccountParameterCatalog, catalog.Name())
+		assertParameterEquals(t, sdk.AccountParameterLogLevel, string(sdk.LogLevelInfo))
+		assertParameterEquals(t, sdk.AccountParameterTraceLevel, string(sdk.TraceLevelOnEvent))
+		assertParameterEquals(t, sdk.AccountParameterReplaceInvalidCharacters, "true")
+		assertParameterEquals(t, sdk.AccountParameterStorageSerializationPolicy, string(sdk.StorageSerializationPolicyCompatible))
+		assertParameterEquals(t, sdk.AccountParameterSuspendTaskAfterNumFailures, "10")
+		assertParameterEquals(t, sdk.AccountParameterTaskAutoRetryAttempts, "10")
+		assertParameterEquals(t, sdk.AccountParameterUserTaskManagedInitialWarehouseSize, string(sdk.WarehouseSizeMedium))
+		assertParameterEquals(t, sdk.AccountParameterUserTaskTimeoutMs, "12000")
+		assertParameterEquals(t, sdk.AccountParameterUserTaskMinimumTriggerIntervalInSeconds, "30")
+		assertParameterEquals(t, sdk.AccountParameterQuotedIdentifiersIgnoreCase, "true")
+		assertParameterEquals(t, sdk.AccountParameterEnableConsoleOutput, "true")
+		assertParameterEquals(t, sdk.AccountParameterPipeExecutionPaused, "true")
+
+		tag1Value, err := client.SystemFunctions.GetTag(ctx, tagTest.ID(), schema.ID(), sdk.ObjectTypeSchema)
+		require.NoError(t, err)
+		assert.Equal(t, "v1", tag1Value)
+
+		tag2Value, err := client.SystemFunctions.GetTag(ctx, tag2Test.ID(), schema.ID(), sdk.ObjectTypeSchema)
+		require.NoError(t, err)
+		assert.Equal(t, "v2", tag2Value)
+	})
 }
 
 func TestInt_SchemasAlter(t *testing.T) {
 	client := testClient(t)
 	ctx := testContext(t)
 
+	assertParameterEquals := func(t *testing.T, params []*sdk.Parameter, parameterName sdk.AccountParameter, expected string) {
+		t.Helper()
+		assert.Equal(t, expected, helpers.FindParameter(t, params, parameterName).Value)
+	}
+
+	assertParameterEqualsToDefaultValue := func(t *testing.T, params []*sdk.Parameter, parameterName sdk.ObjectParameter) {
+		t.Helper()
+		param, err := collections.FindOne(params, func(param *sdk.Parameter) bool { return param.Key == string(parameterName) })
+		assert.NoError(t, err)
+		assert.NotNil(t, param)
+		if param != nil && (*param).Level == "" {
```

Comment:
one more comment here: if we go to else, nothing is validated in this assertion - it is not good, right?

Reply:
Dropped this condition.